izug/refegovservice/upgrades/20191031154412_archetypes_to_dexterity/types.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<object name="portal_types" meta_type="Plone Types Tool">
+  <property name="title">Controls the available contenttypes in your portal</property>
```

Comment:
I suggest removing the old types, e.g.
```xml
  <object name="EgovLeistung" remove="True" />
```

(setting the title of `portal_types` is not really needed here because Plone does that already)

Reply:
✅